contents/docs/hogql/index.mdx

```diff
@@ -26,6 +26,8 @@ SQL insights enables you to create and edit insights using full SQL queries inst
 
 ![HogQL SQL insight](../../images/features/hogql/insights-sql.png)
 
+Read the [full documentation for SQL insights](/docs/product-analytics/sql).
+
```

Comment:
Its quite easy to miss this, since its under an image. So on line 25, I would also add a hyper link there 

"[SQL iinsights]((/docs/product-analytics/sql)) enables you to create and edit insights..."

And then you can probably remove this line